src/bindings/python/tests/test_torchvision_to_ov/test_preprocessor.py

```diff
@@ -38,7 +38,8 @@ def _infer_pipelines(test_input, preprocess_pipeline, input_channels=3):
     ov_model = PreprocessConverter.from_torchvision(
         model=ov_model, transform=preprocess_pipeline, input_example=Image.fromarray(test_input.astype("uint8"), "RGB"),
     )
-    ov_model = core.compile_model(ov_model, "CPU")
+    infer_config = {"INFERENCE_PRECISION_HINT": "f32"}
+    ov_model = core.compile_model(ov_model, "CPU", infer_config)
```

Comment:
```suggestion
    from openvino import Type
    from openvino.properties.hint import inference_precision
    infer_config = {inference_precision: Type.f32}
    # code code code...
    ov_model = core.compile_model(ov_model, "CPU", infer_config)
```
FYI you can use better properties API.

Reply:
Done.